src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceDateInput.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <label>
+    {{ label }}
+    <v-date-input
+      :model-value="new Date(modelValue)"
```

Comment:
for this one, is it possible to use luxon instead of javascript's native Date object? 

Thinking the javascript Date object is annoying to use.

Reply:
I just went with what Vuetify suggested the model-value is for the v-date-input

> The default model value is a Date object, but is displayed as formatted text in the input…
https://vuetifyjs.com/en/components/date-inputs/#model

Maybe there is a way to spit out JS date object from Luxon, do you know?

Reply:
To create one from a standard string format, use [DateTime.fromISO](https://moment.github.io/luxon/api-docs/index.html#datetimefromiso)

Maybe this one?

Reply:
@SoLetsDev tried to use that, same issues with v-date-input throwing console errors. Suggesting we close this and circle back.